daily-patch-information/src/main/java/org/wso2/engineering/efficiency/patch/analysis/email/EmailContentCreator.java

```diff
@@ -0,0 +1,262 @@
+/*
+Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+WSO2 Inc. licenses this file to you under the Apache License,
+Version 2.0 (the "License"); you may not use this file except
+in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+*/
+
+package org.wso2.engineering.efficiency.patch.analysis.email;
+
+import org.wso2.engineering.efficiency.patch.analysis.constants.Constants;
+import org.wso2.engineering.efficiency.patch.analysis.jira.JIRAIssue;
+import org.wso2.engineering.efficiency.patch.analysis.jira.ReportDateComparator;
+import org.wso2.engineering.efficiency.patch.analysis.pmt.InactivePatch;
+import org.wso2.engineering.efficiency.patch.analysis.pmt.OpenPatch;
+import org.wso2.engineering.efficiency.patch.analysis.pmt.Patch;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+import static org.wso2.engineering.efficiency.patch.analysis.constants.Constants.Email.COLUMN_NAMES;
+import static org.wso2.engineering.efficiency.patch.analysis.constants.Constants.Email.COLUMN_NAMES_DEV;
+import static org.wso2.engineering.efficiency.patch.analysis.constants.Constants.Email.COLUMN_NAMES_INACTIVE;
+import static org.wso2.engineering.efficiency.patch.analysis.constants.Constants.Email.COLUMN_NAMES_RELEASED;
+import static org.wso2.engineering.efficiency.patch.analysis.constants.Constants.Email.COLUMN_NAMES_SUMMARY;
+import static org.wso2.engineering.efficiency.patch.analysis.constants.Constants.Email.EMAIL_FOOTER;
+import static org.wso2.engineering.efficiency.patch.analysis.constants.Constants.Email.SECTION_HEADER_DEV;
+import static org.wso2.engineering.efficiency.patch.analysis.constants.Constants.Email.SECTION_HEADER_INACTIVE;
+import static org.wso2.engineering.efficiency.patch.analysis.constants.Constants.Email.SECTION_HEADER_RELEASED;
+import static org.wso2.engineering.efficiency.patch.analysis.constants.Constants.Email.SECTION_HEADER_SIGNING;
+import static org.wso2.engineering.efficiency.patch.analysis.constants.Constants.Email.SECTION_HEADER_SUMMARY;
+
+/**
+ * Creates and returns the body of the email.
+ */
+public class EmailContentCreator {
+
+    private static EmailContentCreator emailContentCreator = new EmailContentCreator();
+
+    private EmailContentCreator() {
+    }
+
+    public static EmailContentCreator getEmailContentCreator() {
```

Comment:
The method should be as 'getInstance()' than repeating the class name. It doesn't look nice to see EmailContentCreator.getEmailContentCreator(). Fix this in all singleton class that you have defined.